bookwyrm/migrations/0173_default_user_auth_group_setting.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.16 on 2022-12-27 21:34
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def backfill_sitesettings(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    group_model = apps.get_model("auth", "Group")
+    editor_group = group_model.objects.using(db_alias).filter(name="editor").first()
+
+    sitesettings_model = apps.get_model("bookwyrm", "SiteSettings")
+    for site in sitesettings_model.objects.all():
```

Comment:
totally not important, but I think you can write this as:
```python
sitesettings_model.objects.update(default_user_auth_group=editor_group)
```

and not use a loop.